config.json

```diff
@@ -675,7 +792,17 @@
         "slug": "protein-translation",
         "name": "Protein Translation",
         "uuid": "84af7be8-8f90-4990-9d48-b5324a0d4354",
-        "prerequisites": [],
+        "prerequisites": [
+          "maps",
+          "enum",
+          "recursion",
+          "case",
+          "pattern-matching",
+          "strings",
+          "binaries",
+          "regular-expressions",
+          "errors"
+        ],
```

Comment:
The example solution to this problem uses metaprogramming like this:
https://github.com/exercism/elixir/blob/main/exercises/practice/protein-translation/.meta/example.ex#L49-L52

I wonder if this should become a concepts of its own? How would it be called, which parts of metaprogramming would it cover?

Reply:
Maybe it could cover `quote` and `unquote` so that those are not new concepts when macros are introduced?